### PR TITLE
Add some more informations in helpers

### PR DESCRIPTION
**Pull request by ksagnard**

### CODE REVIEW

File: helper/src/main.ts [hunk 1]
```diff
@@ -37,9 +37,11 @@ async function run(): Promise<void> {
     core.info('_____________ #> curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"')
     core.info('_____________ #> sudo installer -pkg AWSCLIV2.pkg -target /')
     core.info('_____________ #> aws --version')
-    core.info('(once) Step 2: Configure the AWS CLI with your credentials')
+    core.info('(once) Step 2: Configure the AWS CLI with your credentials and the region of your instance')
+    core.info('those information can be found in your aws account, the region of your instance is not necessarily')
+    core.info('where you currently are (for testing we often use the region Ireland')
```
Comment (Fgerthoffert) [inline helper/src/main.ts:42]: Can you add the name of the region to use in the config ?

File: helper/src/main.ts [hunk 2]
```diff
@@ -37,9 +37,11 @@ async function run(): Promise<void> {
...
     core.info('_____________ #> aws configure')
-    core.info('(once) Step 3: Install Session Manager plugin on your machine')
+    core.info('(once) Step 3: Install Session Manager plugin on your machine, python 2.7 is required for this part')
```
Comment (Fgerthoffert) [inline helper/src/main.ts:44]: I believe it's having the "python" binary which is necessary, not a particular version of python, isn't it?

File: helper/src/main.ts [hunk 3]
```diff
@@ -52,6 +54,9 @@ async function run(): Promise<void> {
     core.info('Step 1: Connect using the AWS CLI')
     core.info('_____________ INSTANCE_ID: AWS EC2 Instance ID, displayed at the top of this job, or via the EC2 console')
     core.info('_____________ #> aws ssm start-session --target INSTANCE_ID')
+    core.info('Step 2: Access the dockers')
```
Comment (Fgerthoffert) [inline helper/src/main.ts:57]: the `containers`

File: helper/src/main.ts (hunk 2 above)
Comment (ksagnard) [reply helper/src/main.ts:44]: with python 3, I got some error while execution the next steps, I have installed python 2 and I worked fine

**Review by Fgerthoffert — APPROVED** [review]

**Pull request merged**